sdk/storage/azure-storage-blob/azure/storage/blob/container_client.py

```diff
@@ -635,7 +635,7 @@ def get_container_access_policy(self, **kwargs):
 
     @distributed_trace
     def set_container_access_policy(
-            self, signed_identifiers=None,  # type: Optional[Dict[str, Optional[AccessPolicy]]]
+            self, signed_identifiers,  # type: Dict[str, AccessPolicy]
```

Comment:
We are using the `signed_identifiers` in the code and we are still guarding it from being `None`. But the check may be incorrect (not because of this change) - what is supposed to happen if you specify an empty set of of signed identifiers?